gulpfile.js

```diff
@@ -46,18 +46,28 @@ function modifyIndex() {
             meta.setAttribute('content', 'default-src * \'self\' \'unsafe-inline\' \'unsafe-eval\' data: gap: file: filesystem: ws: wss:;');
             this.head.appendChild(meta);
 
-            // Search for injected apploader
-            let apploader = this.body.querySelector('script[src*="apploader"]');
+            // Search for injected main.bundle
+            let apploader = this.head.querySelector('script[src^=main]');
```

Comment:
No chance of this being something like `<script src="./main.js" />`?

> `[attr^=value]`
> Represents elements with an attribute name of attr whose value is prefixed (preceded) by value.

> `[attr*=value]`
> Represents elements with an attribute name of attr whose value contains at least one occurrence of value within the string.

- https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors

Reply:
It depends on the webpack configuration: https://github.com/jellyfin/jellyfin-web/blob/051ee1445bd2eb3c57d85474931244e85f33c19e/webpack.common.js#L81-L85
It currently generates `src="main.[hash].bundle.js"`.

Reply:
should be fine then